packages/stream_video_screen_sharing/README.md

```diff
@@ -0,0 +1,15 @@
+# stream_video_screen_sharing
+
+A new Flutter plugin project.
+
+## Getting Started
+
+This project is a starting point for a Flutter
+[plug-in package](https://flutter.dev/developing-packages/),
+a specialized package that includes platform-specific implementation code for
+Android and/or iOS.
+
+For help getting started with Flutter development, view the
+[online documentation](https://flutter.dev/docs), which offers tutorials,
+samples, guidance on mobile development, and a full API reference.
+
```

Comment:
Can we add a small README to this package and also point to the screensharing guide?